classes/options-general.php

```diff
@@ -394,19 +394,19 @@ function print_page() {
 									<!-- #### MAP CONFIGURATION #### -->
 									<div class="postbox">
 
-										<h3><?php _e( 'Map Configuration', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Map Configuration', 'SimpleMap' ); ?></h3>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->